### DISCUSSION TITLE
Upcoming Organized app release: We need your translation help! 🙌

### DISCUSSION THREAD

**Discussion by ux-git** [sws2apps/organized-app#2219] [Announcements]
## The day has come: Organized app release date is around the corner!

We're excited to finally announce that the release of Organized! We have put in a huge amount of effort, time, and sometimes even sleepless nights into making the new application version as good as it can be in terms of user experience, speed, and convenience. We truly believe that Organized will save hundreds of hours for brothers and publishers in your congregation and will surely improve the way to manage congregation responsibilities.

**And now we need your help to make this release the best it can be!** How exactly? By translating the app into the language of your heart ❤️, so that not only you but also thousands of brothers and sisters that speak the same language can benefit from the application. While translation work has been in continuous progress throughout the past year, along with development, our goal now is to complete as many translations as possible in the coming weeks, ensuring all users have an amazing experience right from the very start.

<img width="1177" alt="Screenshot 2024-08-02 at 22 47 52" src="https://github.com/user-attachments/assets/edb9cce8-e1c6-4190-8aed-85c983ec07f7"><br><br>

## How you can help

Now is the perfect time to contribute! Whether you can translate a few words, a few dozen, or even thousands, every bit of help is invaluable. Your contributions will make a significant difference!<br><br>

## Get involved

The best thing is that you don't have to start from scratch! Some languages are already fully translated, some are more than 50% complete, and some have smaller completed translation percentages, but they're not completely empty:

<img width="210" alt="Screenshot 2024-08-02 at 22 48 51" src="https://github.com/user-attachments/assets/36ff8f95-6391-497e-a572-ed48c340ce79">

To participate, head over to our translation platform [Crowdin](https://crowdin.com/project/organized). Simply open the page, find your language, check if there's a need to translate it, and start translating. The translation project includes numerous screenshots that illustrate how and where specific text strings are used, along with comments and contextual explanations.

We also suggest you view our short yet informative [video tutorial](https://www.youtube.com/watch?v=GG5q_NkfD6s) to learn more about the translation workflow and how to make it easier for you. Thank you a lot for helping us make Organized better for everyone!

Best wishes,  
Your Organized app team

**Comment by fschein** [comment]
I'm revising and translating what's missing in Portuguese. I've sent some suggestions for changes to what we usually use here in Brazil.

**Reply by ux-git** [reply]
Thank you so much @fschein 🤝
